submissions/SergeyShytikov/tiny-js-world-task/index.js

```diff
@@ -0,0 +1,68 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Ice',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'woof-woof!'
+};
+const cat = {
+   species: 'cat',
+   name: 'Cloud',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'hsssshhhhh'
+};
+const woman = {
+   species: 'human',
+   name: 'Sara',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi all!'
+};
+const man = {
+   species: 'human',
+   name: 'David',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Hello everyone!'
+};
+const inhabitants = [dog, cat, man, woman];
+
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+/* Print examples:
+   print('ABC');
+   print('<strong>ABC</strong>');
+   print('<strong>ABC</strong>', 'div');
+
+   print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
+   print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
+   */
+
+
+function getPropertiesValueToString(obj) {
+return print(Object.values(obj).join('; '))
+}
```

Comment:
Looks like a generic method to convert any object into a string.

However, as the task requires specific order of properties listed `Object` methods like `keys`, `values`, `entries` shouldn't be used as these do not guarantee any particular order of keys/values.

Once you solve this the method will stop being generic but rather specific to our domain (Tiny JS World and its inhabitants). Thus you will need to reflect this specificity in function and its parameter names.

Reply:
Thanks for the advice, for some reason I did not read all the requirements of the task.